### ISSUE TITLE
Icinga DB: is_reachable and severity may miss updates

### ISSUE THREAD

**Issue by julianbrost** [Icinga/icinga2#9143]
The Icinga DB code never connects to the `Checkable::OnReachabilityChanged` signal, so the `is_reachable` attribute can probably become outdated in Redis and MySQL (haven't explicitly tested for this but I think you should be able to construct a case where this happens).

This also affects the `is_reachable` attribute of services as this one depends on the reachability of its host.

**Comment by Al2Klimov** [comment]
> you should be able to construct a case where this happens

E.g. if parent goes down?

**Comment by julianbrost** [comment]
Yes, that will be a step in trying to reproduce the issue. I just created that issue as a note without explicitly triggering the bug and thus providing precise instructions. Might also be required to have a long check interval or disable active checks to observe a wrong value in Redis or something like that.

**Comment by Al2Klimov** [comment]
Shall we just listen on that signal and make a status update?

**Comment by julianbrost** [comment]
Yes, that should fix it.
